infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -184,6 +187,15 @@ private void processor() {
                                 // Retrieve the address space for every stream in the log.
                                 currOp.setResultValue(streamLog.getStreamsAddressSpace());
                                 break;
+                            case MISSING_ADDRESSES_IN_RANGE:
```

Comment:
This doesn't have to be part of the batch writer, please remove it.

Reply:
Fixed.